egov/egov-egi/src/main/java/org/egov/infra/admin/master/service/UserService.java

```diff
@@ -180,14 +190,72 @@ else if (user.getType().equals(UserType.EMPLOYEE))
         createUserRequest.setRequestInfo(microserviceUtils.createRequestInfo());
 
         final RestTemplate restTemplate = new RestTemplate();
+        UserDetailResponse udr = null;
         try {
-            restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
+            udr = restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
         } catch (final Exception e) {
```

Comment:
That would be better, also is it possible to call this from Controllers instead of service and leave the service unchanged (If possible please do else move it MicroserviceUtils) ?